examples/apps/branching/src/container/runtimeFactory.ts

```diff
@@ -67,20 +36,8 @@ export class GroceryListContainerRuntimeFactory implements IRuntimeFactory {
 				containerRuntime,
 				groceryListId,
 			);
-			const getSuggestions = async () => {
-				const changes = await getChangesFromHealthBot(groceryList);
-				// TODO: Here try integrating staging mode and applying the changes in that mode.
-				// TODO: If we end up providing a way to interrogate local changes, maybe replace the `changes` structure.
-				return {
-					changes,
-					acceptChanges: () => applyDiffToGroceryList(groceryList, changes),
-					rejectChanges: () => {},
-				};
-			};
-			return {
-				groceryList,
-				getSuggestions,
-			};
+			// TODO: Here we could pass in the capability to enter staging mode if it lives on the containerRuntime.
+			return new SuggestionGroceryList(groceryList);
```

Comment:
Any benefit to declaring the return type of the entry point?  i.e. `Promise<FluidObject<ISuggestionGroceryList>>`.  Like if you're searching for usages of that interface you could find it is the benefit I'm thinking of 🤷‍♂️  (it's not like `IRuntime` has a generic parameter or something for it to provide type-safety, more as FYI)